cfpb/consumerfinance.gov
# PfC: Encode URL in financial path tool mailto link

Encode URL in financial path mailto link so all parameters are included.



## Changes

- Encode body of mailto link in financial path tool


## How to test this PR

1. Check out branch and run gulp
2. Navigate to [financial path tool](http://localhost:8000/paying-for-college/your-financial-path-to-graduation/?iped=106397&houp=onCampus&typp=associates&lenp=4&ratp=inState&depp=dependent&houx=1037&fdx=418&clhx=100&trnx=550&hltx=265&entx=140&retx=192&taxx=23&othx=252)
3. On Save & Finish tab, verify that when mailto link is clicked, body of email includes all query params of financial path tool  link


## Screenshots

### Production

<img width="919" alt="Screen Shot 2021-01-19 at 11 58 59 AM" src="https://user-images.githubusercontent.com/778171/105086883-82013c00-5a4e-11eb-9b3e-f830ea2b1d87.png">


### Branch

<img width="913" alt="Screen Shot 2021-01-19 at 11 58 52 AM" src="https://user-images.githubusercontent.com/778171/105086926-947b7580-5a4e-11eb-9375-56407c684ddf.png">
